templates/kommander.yaml

```diff
@@ -25,5 +25,4 @@ spec:
     repo: https://mesosphere.github.io/charts/stable
     version: 0.1.21
   requires:
-    matchLabels:
-      kubeaddons.mesosphere.io/provides: defaultstorageclass
+    defaultStorageClass: true
```

Comment:
why Kommander requires default storage class?

Reply:
+1

Reply:
@DanielMSchmidt do you potentially have any comments on this? Right now we're just looking at potentially changing how this is defined, but while we're here is this actually a requirement for Kommander?

Reply:
Not that I'm aware of, we don't really store anything on disk in kommander

Reply:
Thanks for the update Daniel.

I've created https://jira.mesosphere.com/browse/DCOS-58414 as a bug report to follow up on this, so as to ensure the change is tested in isolation and not added to the scope of this change.